src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksWebServiceCallsServiceImpl.java

```diff
@@ -349,8 +356,18 @@ public int uploadVendorsToPaymentWorks(InputStream vendorCsvDataStream) {
     }
 
     private Response performSupplierUpload(InputStream vendorCsvDataStream) {
-        Invocation request = buildMultiPartRequestForSupplierUpload(getClient(), buildSupplierUploadURI(), vendorCsvDataStream);
-        return request.invoke();
+        Client client = null;
+
+        try {
+            ClientConfig clientConfig = new ClientConfig();
+            clientConfig.register(CuMultiPartWriter.class);
```

Comment:
The CXF removal changed the Client object setup to rely upon a superclass for handling the Client object's lifecycle. (I think this was due to the Response getting closed prematurely with Jersey instead of CXF, if the Client was closed before reading the response.) To keep the configuration consistent, I would suggest replacing this manual Client creation with an override of the appropriate getClient() method.

This class already overrides the getClient() method to call a variant that registers a multi-part handler, you could try changing that override so that it registers CuMultiPartWriter.class instead of MultiPartFeature.class

Reply:
OK, I have adjusted as requested